modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/configuration/OAuth2AuthorizationServerConfiguration.java

```diff
@@ -31,24 +31,6 @@
 )
 public interface OAuth2AuthorizationServerConfiguration {
 
-	@Meta.AD(
-		deflt = "600",
-		description = "oauth2-authorization-server-access-token-duration-description",
-		id = "oauth2.authorization.server.access.token.duration",
-		name = "oauth2-authorization-server-access-token-duration",
-		required = false
-	)
-	public int accessTokenDuration();
-
```

Comment:
TokenProvider is a customization, an addon. Server configuration should not be removed for it.